src/views/RegisterPage/index.js

```diff
@@ -17,19 +25,39 @@ class RegisterPage extends Component {
       confirmpassword: undefined,
       termsandconditions: undefined
     };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
   }
 
   handleChange = event => {
-    const { info, value } = event.target;
-
-    this.setState({
-      [info]: value
-    });
+    this.setState({[event.target.name]: event.target.value});
     console.log(this.state);
   };
 
   handleSubmit(event) {
     event.preventDefault();
+    var attributeList = [];
+    var dataEmail ={
+      Name: 'email',
+      Value: this.state.email
+    };
+    var dataPersonalName ={
+      Name: 'name',
+      Value: this.state.firstname
+    };
+    var dataFamilyName = {
+      Name: 'family_name',
+      Value: this.state.lastname
+    };
+    attributeList.push(dataEmail);
+    attributeList.push(dataPersonalName);
+    attributeList.push(dataFamilyName);
+
+    UserPool.signUp(this.state.firstname, this.state.password, attributeList, null, (err, data) => {
+      if (err) console.error(err);
```

Comment:
can you make it so that if cognito returns an error it gets displayed somewhere in the frontend? just red text underneath the text boxes or something like that, doesn't have to look pretty for now :)